### PR TITLE
refactor(network): change base uri, removing tables endpoint

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary

The `baseURIs` include the `tables/{chainId}` endpoint, which shouldn't really be used. Instead, it should just be the true `/api/v1` endpoint, attached to the correct gateway.

## Details

See `network.ts` and `baseURIs`.

## How it was tested

N/A

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by dtbuchholz** [comment]
Since this PR makes a tweak to `network.ts`, so I figured I'd call the following out in case there's a change we want to make with `TablelandNetworkConfig` types, described [here](https://github.com/tablelandnetwork/tableland-js/pull/36#discussion_r1375145438). TL;DR—the TS compiler doesn't always like the `string | number` type when we use evm-tableland imports in the SDK, which was introduced when we added `validatorPollingTimeouts`. Thus, a `@ts-expect-error` is needed to resolve the issue...not a big deal but just noting the behavior.

**Comment by dtbuchholz** [comment]
Also, @brunocalza—do we _really_ need the `optimism-goerli-staging` definition in these exports? That config info is only really relevant to and is already configured on the validator, right? Or, do we use `@tableland/evm` for internal testing purposes somewhere, too, and need it to exist in this package?

**Comment by dtbuchholz** [comment]
@joewagner I came across the [`changesets`](https://lirantal.com/blog/introducing-changesets-simplify-project-versioning-with-semantic-releases/) tool while making a [PR in D1's ORM repo](https://github.com/Interactions-as-a-Service/d1-orm/pull/71). It's a pretty nice experience because you just run `npx changeset` on your feature branch, which prompts you for major/minor/patch and a description of changes—this gets written to a `.changeset/<file.md>`. Then, when you merge the branch, it'll automatically handle publishing to npm, tags, and can update the package.json's version, too.

If you think it's worthwhile, I can try it out on this repo since the publishing action is broken, anyways. It's also supposed to be nice for monorepos, but I'm not sure if we need that with our lerna setup for the SDK/CLI/LT.

**Comment by brunocalza** [comment]
> Also, @brunocalza—do we _really_ need the `optimism-goerli-staging` definition in these exports? That config info is only really relevant to and is already configured on the validator, right? Or, do we use `@tableland/evm` for internal testing purposes somewhere, too, and need it to exist in this package?
> 
> edit: I suppose we _do_ need it because we use that type def in the [hardhat config](https://github.com/tablelandnetwork/evm-tableland/blob/469f11ab10682eee8ccbfd86a7e8784f176df4dd/hardhat.config.ts#L13), which we obv need in order to deploy to the staging network.

yeah, I guess you're right, we need it for deploying the contract

**Comment by joewagner** [comment]
> @joewagner I came across the [`changesets`](https://lirantal.com/blog/introducing-changesets-simplify-project-versioning-with-semantic-releases/) tool while making a [PR in D1's ORM repo](https://github.com/Interactions-as-a-Service/d1-orm/pull/71). It's a pretty nice experience because you just run `npx changeset` on your feature branch, which prompts you for major/minor/patch and a description of changes—this gets written to a `.changeset/<file.md>`. Then, when you merge the branch, it'll automatically handle publishing to npm, tags, and can update the package.json's version, too.
> 
> If you think it's worthwhile, I can try it out on this repo since the publishing action is broken, anyways. It's also supposed to be nice for monorepos, but I'm not sure if we need that with our lerna setup for the SDK/CLI/LT.

I'm not sure, but I think we tried changesets in the past and didn't end up going with it.  @carsonfarmer would remember better than me.
I'd prefer to just fix the publish action instead of introducing more tooling.  We publish this so infrequently that I've neglected to spend any time trying to fix the publish action.  If you want to work on that, feel free.  In the mean time let's publish this manually when its ready.

File: network.ts [hunk 1]
```diff
@@ -43,26 +43,25 @@ export const proxies: TablelandNetworkConfig = {
   "local-tableland": "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512",
 };
 
-const homesteadURI = "https://tableland.network/api/v1/tables/1/";
```
Comment (sanderpick) [inline network.ts:46]: The chain Ids are used in the ERC721a `tokenURI` method. It simply appends the token / table Id to the baseUri. Wouldn't this change break that functionality?

Comment (dtbuchholz) [reply network.ts:46]: @sanderpick ah, i see. the use case i was considering this for was to be able to set the baseURI via a hardhat script that only uses evm-tableland, so it'd be nice to be able to import the baseURIs from `network.ts`. but, i think the TablelandDeployments [baseURIs](https://github.com/tablelandnetwork/evm-tableland/blob/469f11ab10682eee8ccbfd86a7e8784f176df4dd/contracts/utils/TablelandDeployments.sol#L104) can help there, and instead, the SDK `getBaseUrl` can provide the correct URL info. i'll just close this out.

and @joewagner sgtm, will tal in the future with #414

Comment (sanderpick) [reply network.ts:46]: if all you need is the host name, you could still use these URLs... just pull the host out. `/api/v1` will be the same for them all.

**Pull request closed without merging**